vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -94,14 +115,35 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         return data;
     }
 
+    private List<List<String>> extractAllDataFromRage(IExcelSheetParser sheetParser, String range)
+    {
+        List<List<String>> listList = new ArrayList<>();
+        CellRangeAddress cellRangeAddress = CellRangeAddress.valueOf(range);
+        int sizeColumn = cellRangeAddress.getLastColumn() - cellRangeAddress.getFirstColumn() + 1;
+        int sizeRow = cellRangeAddress.getLastRow() - cellRangeAddress.getFirstRow() + 1;
+        for (int i = 0; i < sizeColumn; i++)
+        {
+            int l = i;
+            List<String> columnList = new ArrayList<>();
+            for (int j = 0; j < sizeRow; j++)
+            {
+                columnList.add(sheetParser.getDataFromRange(range).stream().map(CellValue::getValue)
+                          .collect(Collectors.toList()).get(l));
```

Comment:
looks like the result of the following comment won't be changed, so you can reuse it
```
sheetParser.getDataFromRange(range).stream().map(CellValue::getValue).collect(Collectors.toList())
```